starter-code/iteration-1/main.ts

```diff
@@ -1,6 +1,47 @@
 // Iteration 1
 // 1. Create an array of strings
 
+// Iteration 1
+// 1. Create an array of strings
+let tasks: string[] = [];
+
+// 2. Create a function to add a tasks in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
+
+function addTask(taskToBeAdded: string) {
```

Comment:
You need to add `:number` --> `function addTask(taskToBeAdded: string):number {`